runtime/src/accounts_background_service.rs

```diff
@@ -100,6 +101,9 @@ impl SnapshotRequestHandler {
         self.snapshot_request_receiver
             .try_iter()
             .last()
+            .filter(|_snapshot_request| {
```

Comment:
This is the code where we GENERATE snapshots. We want to always generate snapshots. The check does not go here, I do not think.